src/components/Chat/Chat.tsx

```diff
@@ -159,8 +161,16 @@ export const Chat = memo(() => {
           }
           return !modelIds.includes(conv.model.id);
         });
-    setIsNotAllowedModel(isNotAllowed);
-  }, [selectedConversations, models, modelsIsLoading]);
+    if (isNotAllowed) {
+      setNotAllowedType(EntityType.Model);
+    } else if (
+      selectedConversations.some((conversation) =>
```

Comment:
nit: could you create separate variable for isNotAllowedAddon and name previous one to isNotAllowedModel